content/practices/color-settings/color-settings-practice.html

```diff
@@ -0,0 +1,1568 @@
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <code>prefers-color-scheme: dark</code>
+              </td>
+              <td>
+                <img style="width: 95%" src="./images/wikipedia-scheme-dark.png" alt="Screen shot of wikipedia page with user preferences sidebar open.  The red circle shows the users settings radio button for dark settings checked.">
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <h3>Testing Dark Color Scheme</h3>
+        <p>In  each operating system used by the target audience:</p>
+        <ol>
+          <li>Turn on the dark color theme.</li>
+          <li>
+            Verify contrast for text and components meet or exceed the contrast ratios specified by
+            <a href="https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum">WCAG 1.4.2: Contrast (Minimum)</a>.
+          </li>
+        </ol>
+      </section>
+
+      <section id="forced-colors">
+        <h2>Contrast Themes (Forced Colors)</h2>
+
+        <p>
+          Contrast themes are an operating system accessibility feature that automatically forces all native apps and web content to render using an alternative color theme chosen by the user.
+          Some of the themes are designed to provide extra high contrast, and users can customize their chosen color theme.
+          Browsers automatically replace each author specified color with one of a small set of colors specified by the user's chosen theme based on the type of HTML element.
+          The replacement color used by a browser for a specific element usually corresponds to one of the system colors described below.
+        </p>
+        <p>
+          Note that ARIA attributes do not inform replacement color selection, e.g., a <code>div</code> with role <code>button</code> renders with text colors, not button colors.
+          When the user chooses a contrast theme, such as described by
+          <a href="https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696">Windows 11 Accessibility Settings &gt; Contrast Theme</a>,
+          browsers set the <code>forced-colors</code> property to <code>active</code>.
+          When using ARIA to alter the semantics of elements, specify system colors that match the semantics.
+          For example text used in custom components for labels and instructions should use the same colors as the users theme for text content.
+          Components that behave as links need to match the theme styling for links and interactive components should be based on the styling used for buttons, textbox and range controls for their features.
+        </p>
+
+        <p>
+          Authors can use either <code>currentcolor</code> or <code>&lt;system-colors&gt;</code> CSS values for components to adapt to appropriate colors in the contrast theme.
```

Comment:
This is the first mention of currentcolor. It seems out of place. At this point, we have some explanation of the fact that the browser uses system colors for replacement colors, but no understanding of how currentcolor is relevant to authors. We don't have any explanation of currentcolor, and we don't seem to have a link to MDN for it.